BrainPortal/app/models/portal_task.rb

```diff
@@ -545,52 +545,71 @@ def params_path_value(paramspath) #:nodoc:
     foundvalue
   end
 
-  # Wrapper class around the ActiveRecord errors() method;
-  # this class answers to the same methods as the errors
+  # Wrapper around the ActiveModel::Errors class.
```

Comment:
I think we're missing some methods still... check with

http://api.rubyonrails.org/classes/ActiveModel/Errors.html

For instance, `generate_message` is gone ?